ecs-cli/modules/clients/aws/cloudformation/params_test.go

```diff
@@ -91,6 +85,7 @@ func TestAddWithUsePreviousValueAndValidate(t *testing.T) {
 	}
 
 	for _, param := range params {
+		fmt.Println(param)
```

Comment:
Why is this being printed? I'm guessing it was for debug?

Reply:
....crap.